src/win/sugar.rs

```diff
@@ -47,12 +47,7 @@ impl TryFrom<Vec<u16>> for OSUnicodeString {
         let mut content = content;
         content.push(0);
         let mut inner = MaybeUninit::uninit();
-        unsafe {
-            NTStatusError::from(RtlInitUnicodeStringEx(
-                inner.as_mut_ptr(),
-                content.as_mut_ptr(),
-            ))
-        }?;
+        unsafe { NTStatusError::from(init_unicode_string(inner.as_mut_ptr(), &mut content)) }?;
         // The manual copying of fields is because RtlInitUnicodeStringEx is
         // working on the winapi type definition.
```

Comment:
I think this comment can go

Reply:
Actually we can just remove the manual copying of fields. Not needed anymore.